hazelcast/src/main/java/com/hazelcast/query/impl/getters/CompactGetter.java

```diff
@@ -33,7 +33,7 @@
     @Override
     Object getValue(Object target, String fieldPath) throws Exception {
         InternalGenericRecord record;
-        if (target instanceof CompactInternalGenericRecord) {
+        if (target instanceof CompactGenericRecord) {
```

Comment:
I see that we will use this getter if `targetObject instanceof CompactGenericRecord` (from `Extractors#L148`). So, how come this if will return false we hit the else branch?